security/authorization/engine/util.go

```diff
@@ -61,3 +61,8 @@ func compileStringToCheckedExpr(expr string, declarations []*expr.Decl) (*expr.C
 	}
 	return checkedExpr, nil
 }
+
+func compileStringToExpr(expr string, declarations []*expr.Decl) *expr.Expr {
+	checkedExpr, _ := compileStringToCheckedExpr(expr, declarations)
```

Comment:
We should handle the error here.

Reply:
Ok, I'll log it since returning the error would make the usage cases in the tests much more complicated (need an extra line of code for each use case).